# Type: docs
# Subtype: D2
# English Summary: Add comments
# Chinese Summary: 添加注释
# Entry Number: 16

# === Git Diff Content ===

diff --git a/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/UserInfoRestTemplateConfiguration.java b/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/UserInfoRestTemplateConfiguration.java
index 94b4677..cc7922f 100644
--- a/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/UserInfoRestTemplateConfiguration.java
+++ b/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/UserInfoRestTemplateConfiguration.java
@@ -31,6 +31,9 @@ import org.springframework.security.oauth2.client.token.grant.code.Authorization
 import org.springframework.util.CollectionUtils;
 
 /**
+ * Configuration acting as a factory for the rest template used for extracting user info
+ * during authentication.
+ *
  * @author Dave Syer
  */
 @Configuration
@@ -89,4 +92,4 @@ public class UserInfoRestTemplateConfiguration {
 		return new OAuth2RestTemplate(details, this.oauth2ClientContext);
 	}
 
-}
\ No newline at end of file
+}
